test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -594,6 +598,7 @@ class ExtProcIntegrationTest : public HttpIntegrationTest,
   TestScopedRuntime scoped_runtime_;
   std::string header_raw_value_{"false"};
   std::string filter_mutation_rule_{"false"};
+  bool downstream_filter_{true};
```

Comment:
nit: I would suggest we add this flag to struct `ConfigOptions` . This is more consistent with current code on how to provide the customized configuration options to `initializeConfig()` . It is also good to put those options in one place

Reply:
Thanks for reviewing!
Addressing this in : https://github.com/envoyproxy/envoy/pull/33511